rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMInitialState.java

```diff
@@ -175,32 +172,32 @@ public RMInitialState cloneAndFilter(long filter) {
                                       "Probably because there was network server restart.",
                                       filter,
                                       latestCounter.get()));
-            actualFilter = EMPTY_STATE; // reset filter to default  value
+            actualFilter = -1; // reset filter to default  value
         }
         RMInitialState clone = new RMInitialState();
 
-        clone.nodeEvents = newFilteredEvents(this.nodeEvents,
-                                             actualFilter,
-                                             actualFilter + PAResourceManagerProperties.RM_REST_MONITORING_MAXIMUM_CHUNK_SIZE.getValueAsInt());
-        clone.nodeSourceEvents = newFilteredEvents(this.nodeSourceEvents,
-                                                   actualFilter,
-                                                   actualFilter + PAResourceManagerProperties.RM_REST_MONITORING_MAXIMUM_CHUNK_SIZE.getValueAsInt());
+        clone.nodeEvents = newFilteredEvents(this.nodeEvents, actualFilter);
+        clone.nodeSourceEvents = newFilteredEvents(this.nodeSourceEvents, actualFilter);
 
         clone.latestCounter.set(Math.max(actualFilter,
                                          Math.max(findLargestCounter(clone.nodeEvents.values()),
                                                   findLargestCounter(clone.nodeSourceEvents.values()))));
         return clone;
     }
 
-    private <T extends RMEvent> Map<String, T> newFilteredEvents(Map<String, T> events, long from, long to) {
+    private <T extends RMEvent> Map<String, T> newFilteredEvents(Map<String, T> events, long filter) {
         return events.entrySet()
                      .stream()
-                     .filter(entry -> from < entry.getValue().getCounter() && entry.getValue().getCounter() <= to)
-                     .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+                     .filter(entry -> entry.getValue().getCounter() > filter)
+                     .collect(Collectors.toMap(entry -> entry.getKey(), entry -> entry.getValue()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)